src/hooks/core/useCoreChatSystemHook.ts

```diff
@@ -274,6 +275,8 @@ const useCoreChatSystemHook = () => {
         refresh('channelMember');
       }
     });
+
+    updateAppBadgeFromDB(localDb);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `updateAppBadgeFromDB` function is called with `localDb` as an argument. Make sure that `localDb` is defined and valid at this point in the code. Also, ensure that `updateAppBadgeFromDB` handles potential errors to avoid crashes.
```javascript
updateAppBadgeFromDB(localDb);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->